mapiproxy/servers/default/emsmdb/oxomsg.c

```diff
@@ -121,7 +121,8 @@ static void oxomsg_mapistore_handle_message_relocation(struct emsmdbp_context *e
 			continue;
 		}
 
-		/* FIXME: (from oxomsg 3.2.5.1) PidTagMessageFlags: mfUnsent and mfRead must be cleared */
+		/* FIXME: (from oxomsg 3.2.5.1) PidTagMessageFlags: mfUnsent and mfRead must be cleared.
+		   Note: Property not managed by any current mapistore backend */
```

Comment:
I put it to prevent an incautious developer like me from willing to fix it to fix it.